docs/StyleGuide.md

```diff
@@ -701,14 +934,44 @@ double barCm = 200 * cm;
 ...
 ``` 
 
+You can output the data on the unit you wish. To do so it is sufficient to 
+DIVIDE the data by the corresponding unit:
+
+```cpp     
+G4cout << KineticEnergy/keV << " keV" ;
+G4cout << density/(g/cm3)   << " g/cm3" ;
+```
+(of course: G4cout << KineticEnergy will print the energy in the internal system 
+of units)
+
+
+There is another way to output the data; lets Geant4 chooses the most approriate 
```

Comment:
"There is another way to output data which lets Geant4 choose the most appropriate"